libkbfs/md_ops.go

```diff
@@ -46,12 +46,19 @@ func (md *MDOpsStandard) convertVerifyingKeyError(ctx context.Context,
 }
 
 func (md *MDOpsStandard) verifyWriterKey(
-	ctx context.Context, rmds *RootMetadataSigned) error {
+	ctx context.Context, rmds *RootMetadataSigned, isTlfFinal bool) error {
 	if !rmds.MD.IsWriterMetadataCopiedSet() {
-		err := md.config.KBPKI().HasVerifyingKey(ctx,
-			rmds.MD.LastModifyingWriter,
-			rmds.MD.WriterMetadataSigInfo.VerifyingKey,
-			rmds.untrustedServerTimestamp)
+		var err error
+		if isTlfFinal {
+			err = md.config.KBPKI().HasUnverifiedVerifyingKey(ctx,
```

Comment:
But when we ask for the list of unverified keys from the service, does it give us ALL the keys, even verified ones?  Or just the unverified ones?  If the latter, then this check won't pass if the key is verified.